app-i18n/koreanim/koreanim-0.0.1.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Korean Input Method for BeOS/Haiku"
+DESCRIPTION="Korean Input Method for BeOS/Haiku using libhangul"
+HOMEPAGE="https://github.com/terrayi/KoreanIM"
+COPYRIGHT="2014 Terra Yi"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/terrayi/KoreanIM/archive/refs/tags/v0.0.1.zip"
+SOURCE_DIR="KoreanIM-$portVersion"
+CHECKSUM_SHA256="3af306a4bb5db591b60105e0f98e24f7956a748abe7170e0cf7760ffbf2bbd6b"
```

Comment:
SOURCE_DIR after CHECKSUM_SHA256 (check guidelines)

Reply:
That is wha I have missed! I have only looked at BUILD_* etc but not those meta info(?)